src/optimizer/plugins/ResolveValueSetComponentRuleURLsOptimizer.ts

```diff
@@ -0,0 +1,41 @@
+import { utils } from 'fsh-sushi';
+import { OptimizerPlugin } from '../OptimizerPlugin';
+import { Package } from '../../processor';
+import { resolveURL } from '../utils';
+import {
+  ExportableValueSetConceptComponentRule,
+  ExportableValueSetFilterComponentRule
+} from '../../exportable';
+
+export default {
+  name: 'resolve_value_set_component_rule_urls',
+  description:
+    'Replace URLs in value set rules with their names (for local and FHIR Core URLs only)',
+
+  optimize(pkg: Package, fisher: utils.Fishable): void {
+    [...pkg.valueSets].forEach(vs => {
```

Comment:
This can just be `pkg.valueSets.forEach`

Reply:
Ha.  Of course.  Silly me.

Reply:
Fixed in c6aede1